packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/backup.constants.js

```diff
@@ -2,13 +2,22 @@ export const BACKUP_OFFER_LEGACY = 'legacy';
 export const BACKUP_OFFER_PREMIUM = 'premium';
 export const BACKUP_OFFER_ADVANCED = 'advanced';
 export const BACKUP_OFFER_CLASSIC = 'classic';
-
+export const BACKUP_OFFERS = ['classic', 'advanced', 'premium'];
```

Comment:
Looks like the offers are right above this line of code

Reply:
You shouldn't hardcode data available in api schema. 
https://api.ovh.com/1.0/dedicatedCloud.json exposes the enum you need : 

```json
  "dedicatedCloud.backup.OfferTypeEnum": {
    "id": "OfferTypeEnum",
    "enumType": "string",
    "description": "All the offer type of the backup",
    "namespace": "dedicatedCloud.backup",
    "enum": [
      "advanced",
      "backup",
      "classic",
      "legacy",
      "premium"
    ]
  },
```